### ISSUE TITLE
CookieService: Method Laminas\Session\Config\ConfigInterface::getCookieSameSite does not exist

### ISSUE THREAD

**Issue by alexmerlin** [dotkernel/frontend#347]
```
Error: src/App/src/Service/CookieService.php:69:73: UndefinedInterfaceMethod: Method Laminas\Session\Config\ConfigInterface::getCookieSameSite does not exist (see https://psalm.dev/181)
```

**Comment by alexmerlin** [comment]
The method actually exists... I'll fix this.

**Comment by alexmerlin** [comment]
Fixed in #348
